elements/cc-ubuntu24.04-rocm/package-installs.yaml

```diff
@@ -0,0 +1,17 @@
+linux-generic-hwe-24.04:
+linux-headers-generic-hwe-24.04:
+linux-modules-extra-6.11.0-19-generic:
```

Comment:
I believe that this is pulled in as a dependency, the reason to not install manually being in the case where ubuntu updates the minor release version.

- linux-generic-hwe-24.04
  - linux-headers-generic-hwe-24.04
    - linux-headers-6.11.0-19-generic
  - linux-image-generic-hwe-24.04
    - amd64-microcode
    - intel-microcode
    - linux-firmware
    - linux-image-6.11.0-19-generic
    - linux-modules-extra-6.11.0-19-generic